README.md

```diff
@@ -56,7 +56,7 @@ It is generally safe and recommended to use the master branch.
 Add this line to your application's Gemfile:
 
 ```
-gem 'active_model_serializers'
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
seems like a good idea. There's been discussion of doing something similar to what Rake and Node.js have done and jump to `10.0` (I'm for it), but it hasn't been resolved yet.  pre-1.0 semver is weird.

@krisleech Thanks for sharing this PR.  Anything else you might contribute back to help out others is much appreciated!